modules/enabling-scanner-v4-operator.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-operator_{context}"]
+= Enabling RHACS Scanner V4 when installing with the Operator
+
+[role="_abstract"]
+Scanner V4 is not enabled by default, but you can enable it during installation.
+
+.Procedure
+
+. To enable Scanner V4 on the cluster where Central is installed, follow the installation procedures as described in "Installing Central using the Operator method". Choose one of these methods:
+* As described in that procedure, when configuring the available options for Central, go to the *Scanner V4 Component Settings* section and in the *Scanner V4 Component* menu, select *Enabled*.
+* In the Central custom resource (CR) YAML, configure the following parameter:
++
+[source,yaml]
+----
+  scannerV4:
+    scannerComponent: Enabled
+----
+. After you enable Scanner V4 on the cluster where Central is installed, you configure your init bundle or cluster registration secret (CRS) to allow communication between Central and the secured cluster. Then, you can enable Scanner V4 on the secured cluster by following the installation procedures as described in "Installing {product-title-short} on secured clusters by using the Operator". Choose one of these methods:
```

Comment:
I think this is a little much for me to unpack and provide a good suggestion, but I just want to note that per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps), the first sentence of each command must be written in the imperative mood (so that it's written as a command and has an instruction to follow). Could this be rewritten/restructured to accommodate that?

Reply:
So ideally the customer would be using the installation documentation to install (and enabling Scanner V4 information is there) - so this is more of a summary of steps that need to be done, rather than a procedure. Maybe I should just make this a concept and make that more clear?